gimmemotifs/scanner.py

```diff
@@ -494,14 +494,14 @@ def scan_sequence(
             if zscore:
                 m_mean, m_std = motifs_meanstd[seq_gc_bin][motif.id]
                 result = pwmscan(
-                    seq, motif.logodds, motif.pwm_min_score(), nreport, scan_rc
+                    seq, motif.logodds.tolist(), motif.min_score, nreport, scan_rc
```

Comment:
Not sure if you're willing to dig into the C source. But since the logodds are now anyway numpy arrays, it seems like a waste to convert it to a list, and then in the C code convert the list into C arrays. No clue, but **perhaps** a numpy array can be read directly by the C code?